recipes/tassel/3.0.137/meta.yaml

```diff
@@ -0,0 +1,24 @@
+{% set name = "TASSEL" %}
+{% set version = "3.0.137" %}
```

Comment:
Are there any reasons to use this specific version?
Looks like `V3.0.174` is the latest: https://sourceforge.net/p/tassel/tassel3-standalone/ref/master/tags/

Reply:
It was a mistake.  I have revised the recipe to build for V3.0.174 in the latest commit.